src/bindgen/config.rs

```diff
@@ -609,6 +609,8 @@ pub struct Config {
     pub namespace: Option<String>,
     /// An optional list of namespaces. Only applicable when language="C++"
     pub namespaces: Option<Vec<String>>,
+    /// An optional list of namespaces to ddeclare as using. Only applicable when language="C++"
+    pub using: Option<Vec<String>>,
```

Comment:
If not, can we at least call this `using_namespaces` or something? Otherwise the new option name makes little intuitive sense to me.